src/components/Page.js

```diff
@@ -0,0 +1,38 @@
+const Page = ({ page }) => {
+  const { pageid, title, description, thumbnail, coordinates } = page;
+  const dist = coordinates[0] === undefined ? 0 : coordinates[0].dist;
+
+  return (
+    <li className="page">
+      <div className="page__info">
+        {thumbnail ? (
```

Comment:
i think this can be simplified a little. in both cases you could use the same `<figure>` pattern and just set different info for the image and `alt` attribute. let's talk about it in office hours!

Reply:
I removed redundant code, thank you for your suggestion. It's valuable.

![image](https://user-images.githubusercontent.com/8689897/126823322-ddc25e91-2f72-49f5-9aac-01ed2756fc4c.png)